### DISCUSSION TITLE
Has Vercel banned China?

### DISCUSSION THREAD

**Discussion by TomsProject** [vercel/community#806] [Help] [answered]
I'm a user from China and today while I tried to access my Vercel dashboard, it says "vercel.com refused the connection".Has Vercel banned China?

**Comment by hallee9000** [comment]
I think it's banned by GFW

**Comment by fu-sen** [comment, marked as answer]
This has been addressed, but if you still have problems please try the status information.
https://www.vercel-status.com/incidents/r758bhbklgfd

**Reply by lxchinesszz** [reply]
Thank you very much。

**Comment by huayemao** [comment]
vercel.app can't be accessed from china again since yesterday, hope someone can help 🥺

**Reply by RavelloH** [reply]
Now vercel.com is OK. You can bind your own domain name and resolve CNAME records to vercel-dns.com

**Comment by feavel1** [comment]
Can't access my web vercel.app pages from china.
Sept 2022.

**Comment by TomsProject** [comment]
*.vercel.app's DNS is being cached poisoning by China's GFW, the only way to solve it now is to bind your own domain.
